.kibana_items.json

```diff
@@ -672,15 +654,6 @@
     {
         "_id": "[TIMINGS]-Top-20-External-Domains",
         "_type": "visualization",
-        "_source": {
-            "title": "[TIMINGS-RES] Top 20 External Domains",
-            "visState": "{\"title\":\"[TIMINGS-RES] Top 20 External Domains\",\"type\":\"pie\",\"params\":{\"shareYAxis\":true,\"addTooltip\":true,\"addLegend\":true,\"legendPosition\":\"right\",\"isDonut\":false},\"aggs\":[{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"schema\":\"metric\",\"params\":{}},{\"id\":\"2\",\"enabled\":true,\"type\":\"terms\",\"schema\":\"segment\",\"params\":{\"field\":\"uri_host\",\"size\":20,\"order\":\"desc\",\"orderBy\":\"1\",\"customLabel\":\"Domains\"}}],\"listeners\":{}}",
-            "uiStateJSON": "{}",
-            "description": "",
-            "version": 1,
-            "kibanaSavedObjectMeta": {
-                "searchSourceJSON": "{\"index\":\"cicd-resource*\",\"query\":{\"query_string\":{\"query\":\"!uri_host:*godaddy* AND !uri_host:*wsimg.com*\",\"analyze_wildcard\":true}},\"filter\":[]}"
-            }
-        }
+        "_source": "delete_this"
```

Comment:
With these 3 marked as "delete_this", would it make sense to just delete them? Or is this more of an instruction for the end user?

Reply:
It's an instruction to go remove them from existing installs...